source/math/IntRandomGenerator.ooc

```diff
@@ -15,17 +15,35 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 use ooc-math
+import math
 import os/Time
 
-IntRandomGenerator: class {
-	_backend: FloatRandomGenerator
-	init: func ~withBackend (=_backend)
-	free: override func {
-		this _backend free()
-		super()
+IntUniformRandomGenerator: class {
+	_state, _min, _max, _range: Int
+	init: func (seed := Time microtime()) {
+		this _min = 0
+		this _max = 32767
```

Comment:
> Suggestion: create an extension in sdk/math.ooc for Short and set 0x7FFF as max value.

```ooc
extend Short {
    maximum: static SHRT_MAX
...
```
would be nice, yeah. But, if `this _min` is 0 and `this _range` is `32767` by default, and the result of the default `next` function returns `this _min + (result % this _range)`, doesn't that mean that the returned value will always be < 32767?